### PR TITLE
Various dart: search styling fix

### PR DESCRIPTION
**Pull request by whimsicallyson**
## Links
* various-dart.glitch.me
* https://github.com/FogCreek/Glitch-Community-Work/issues/495

## Changes:
* Updated styling for filters on user page
* Also updated search in add-to-collection, and hex input for collections

## How To Test:
* Check out user page, add-to-collection pop, and collection color pop, make sure they look correct

## Feedback I'm looking for:
* Are there any additional I missed?

### CODE REVIEW

File: src/components/collection/edit-collection-color-pop.styl [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .container
   width: 154px
+  span
+    border-radius: 3px
+    *    
+      font-family: sansserif
```
Comment (modernserf) [inline src/components/collection/edit-collection-color-pop.styl]: you probably want `font-family: var(--fonts-sans)` for these

**Comment by ttseng** [comment]
styling looked fixed!  while we're add it, can awe change the placeholder (in /components/search-form/index.js) to 'apps, users, teams' (instead of 'bots, apps, users')?  definitely have received feedback on multiple occasions that 'bots' is confusing.

File: src/components/collection/edit-collection-color-pop.styl (hunk 1 above)
Comment (whimsicallyson) [reply src/components/collection/edit-collection-color-pop.styl]: changed!

**Comment by whimsicallyson** [comment]
updated the placeholder text and CSS var!

**Pull request merged**